packages/EasyUtils/src/SensitiveData/StringSanitizers/CreditCardNumberStringSanitizer.php

```diff
@@ -17,8 +19,14 @@ public function sanitizeString(string $string, string $maskPattern, array $keysT
             return $string;
         }
 
+        $creditCardNumberValidator = new CreditCardNumberValidator();
```

Comment:
We can, but it'll break the code in https://github.com/eonx-com/easy-monorepo/blob/master/packages/EasyUtils/src/Bridge/Laravel/EasyUtilsServiceProvider.php#L79 because currently all sanitizers have same constructor. 

Thinking about the best way to solve this. WDYT?

Reply:
done